openforis/collect-earth-online
# CEO-425 Add a loading image for GEE requests.

## Purpose
Adds an image that says "Source Loading..." when making a GEE request. Deletes the unused `hidefilter.png`.

## Related Issues
Closes CEO-425

## Submission Checklist

## Testing
1. Create a new project.
2. Make sure that project has Sentinel imagery enabled. 
3. Go to collect on that project and change some of the parameters for the Sentinel imagery.
4. You should see the "Source Loading..." image before each tile loads.

## Screenshots
![Screenshot from 2022-01-25 17-08-38](https://user-images.githubusercontent.com/40574170/151086289-70e7951f-77d8-43a0-a0a5-e9b8fe6d64eb.png)
